packages/google_sign_in/google_sign_in/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
-## NEXT
+## 6.1.0
 
+* Exposes the new method `canAccessScopes`.
+* Updates example app to separate Authentication from Authorization for those
+  platforms where scopes are not automatically granted upon signIn (like the web).
+  * By popular demand: `signInSilently` returns a User object again on the web.
```

Comment:
/cc @stuartmorgan should I bump the major version again because of this change?

Reply:
What are the implications to client code? If it's just something that was null no longer being null, that doesn't seem breaking.

Reply:
Yeah, this is the method that stopped returning something, but now it does return something because you can check if the user that was returned is authorized to do stuff or not.

You're right that this is not breaking this time, the breaking change in the web plugin was so people didn't unexpectedly get the new behavior without having the methods from this package.